runtime/ftplugin/gitconfig.vim

```diff
@@ -13,3 +13,29 @@ setlocal formatoptions-=t formatoptions+=croql
 setlocal comments=:#,:; commentstring=;\ %s
 
 let b:undo_ftplugin = "setl fo< com< cms<"
+
+if !has('unix')
+  finish
+endif
+
+if !has('gui_running')
+  command! -buffer -nargs=1 Sman
```

Comment:
Nit: the `!` is only necessary to overwrite previously-defined commands. I don't think that's necessary here.